index.js

```diff
@@ -198,15 +198,15 @@ function requestAsEventEmitter(opts) {
 			});
 		});
 
-		cacheReq.on('error', err => {
+		cacheReq.once('error', err => {
```

Comment:
`cacheable-request` could potentially emit more than one error. These should be passed through.